actors/miner/src/ext.rs

```diff
@@ -101,6 +101,7 @@ pub mod power {
         pub quality_adj_power: StoragePower,
         pub pledge_collateral: TokenAmount,
         pub quality_adj_power_smoothed: FilterEstimate,
+        pub epochs_since_ramp_start: i64
```

Comment:
This should be ramp start epoch. You can compute the number of epochs since the ramp start by subtracting the current epoch by the start epoch.